pkgs/by-name/ip/iproute2/package.nix

```
@@ -87,6 +88,10 @@ stdenv.mkDerivation rec {
     "CONFDIR=$(out)/etc/iproute2"
   ];
 
+  postInstall = ''
+    moveToOutput sbin/routel "$scripts"
```

This time I actually made sure it works:
```suggestion
  postFixup = ''
    moveToOutput bin/routel "$scripts"
```
I assume the file `$out/bin/routel` is only created in the fixupPhase, but I'm not sure why.